x-pack/plugins/stack_connectors/server/connector_types/cases_webhook/service.ts

```diff
@@ -113,10 +116,28 @@ export const createExternalService = (
         configurationUtilities,
         'Get case URL'
       );
+
+      const json =
+        getIncidentMethod === WebhookMethods.POST && getIncidentJson
+          ? renderMustacheStringNoEscape(getIncidentJson, {
+              external: {
+                system: {
+                  id: JSON.stringify(id),
+                },
+              },
+            })
+          : null;
+
+      if (json !== null) {
```

Comment:
Shouldn't this be

```suggestion
      if (getIncidentMethod === WebhookMethods.POST && json !== null) {
```

Reply:
I am checking this at line 120:
```typescript
 const json = getIncidentMethod === WebhookMethods.POST && getIncidentJson 
   ? renderMustacheStringNoEscape() 
   : null
``` 
So `json` will be null if `getIncidentMethod` is not `WebhookMethods.POST`